CHANGELOG.md

```diff
@@ -1,3 +1,461 @@
+## [Talos 0.9.0](https://github.com/talos-systems/talos/releases/tag/v0.9.0) (2021-03-22)
+
+Welcome to the v0.9.0 release of Talos!
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/talos-systems/talos/issues.
+
+### New Features
+
+* Control Plane as Static Pods
+* ECDSA Keys for Kubernetes PKI
+* Disk Encryption
+* Virtual Shared IP for Control Plane Endpoint
+
+More in the [docs](https://www.talos.dev/docs/v0.9/introduction/what-is-new/).
+
+
+### Upgrading
+
+Please read [upgrade notes](https://www.talos.dev/docs/v0.9/guides/upgrading-talos/#upgrading-from-talos-08) before upgrading from Talos 0.8.
+
+
+### Contributors
+
+* Andrey Smirnov
+* Artem Chernyshev
+* Alexey Palazhchenko
+* Andrew Rynhard
+* Spencer Smith
+* Seán C McCord
+* Andrew Rynhard
+* Brandon McNama
+* Guilhem Lettron
+* Willem Monsuwe
+* vlad doster
```

Comment:
Sir we capitalize this name?

Reply:
it's going from `git`, so whatever he thinks his name is, it's automated fully. we could use `.mailmap` to do such aggregations in the future